windows-gaming-driver/src/setup/wizard.rs

```diff
@@ -249,12 +325,62 @@ impl<'a> Wizard<'a> {
             println!("");
         }
 
-        // TODO: Add a udev picker step here (just like the one we have for gpu) that selects USB keyboard and mouse.
-        // Then save that to the config and create a udev rule file that runs setfacl so the vfio group can use them.
+        // select mouse and keyboard
+        println!("Step 2: Select USB Devices");
+        println!("To enable input devices we use USB passthrough triggered by keyboard shortcuts.");
+        println!();
+        if !machine.usb_devices.is_empty() {
+            println!("You have currently selected the following usb devices: ");
+            for &(vendor, product) in machine.usb_devices.iter() {
+                println!("    {:04x}:{:04x}", vendor, product);
```

Comment:
???

like

we even have a function to resolve this

the hwid thing

I don't understand why you would print this at all

but if you insist

then at least print names else no one can read this mess